client/ayon_flame/plugins/publish/extract_product_resources.py

```diff
@@ -253,6 +256,8 @@ def process(self, instance):
                 exporting_clip.name.set_value("{}_{}".format(
                     folder_path, segment_name))
 
+            color_out = exporting_clip.get_colour_space()
+            self.log.debug(color_out)
```

Comment:
```suggestion
            flame_colour = exporting_clip.get_colour_space()
            self.log.debug(flame_colour)
            imageio_colorspace = get_remapped_colorspace_from_native(
                flame_colour,
                host_name,
                host_imageio_settings,
            )
            self.log.debug(imageio_colorspace)
```

Reply:
this way we will also add remapping feature for conveting native flame color name onto ocio colorspace name space